src/components/steps/step-sheet/DetailsTab.tsx

```diff
@@ -112,7 +148,7 @@ export function StepDetailsTab({ stepId, runId }: Props) {
 								</Tag>
 							}
 						/>
-						{pipelineRunData.body?.code_reference && (
+						{pipelineRunData.body?.code_reference && repositoryMetadata && (
```

Comment:
As we're loading some new data here, we need a loading skeleton if the data comes in after the rest I think